src/base/utils/mac_utils.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2015  Vladimir Golovnev <glassez@yandex.ru>
```

Comment:
You could put your name & email here if you want.
Or just "The qBittorrent project"